diff --git main/Sky Atlas.md ozone-executor-agent-artifact/Sky Atlas.md
index 149351b..7c0dc52 100644
--- main/Sky Atlas.md	
+++ ozone-executor-agent-artifact/Sky Atlas.md	
@@ -51026,15 +51026,681 @@ Operational GovOps for Operational Executor Agent Amatsu is Soter Labs.
 
 #### A.6.1.2.2 - Operational Executor Agent Ozone [Core]  <!-- UUID: 565660dd-7850-4c3a-8dba-554542bf103a -->
 
-The Agent Artifact for Ozone will be specified in a future iteration of the Atlas. In the interim, the subdocuments herein contain key information regarding Operational Executor Agent Ozone.
+The documents herein specify the logic for Operational Executor Agent Ozone.
 
-##### A.6.1.2.2.1 - Operational Executor Facilitator [Core]  <!-- UUID: d282ccb9-82f8-46da-9180-e15e5714bb88 -->
+##### A.6.1.2.2.1 - Introduction [Core]  <!-- UUID: d282ccb9-82f8-46da-9180-e15e5714bb88 -->
 
-The Operational Facilitator for Operational Executor Agent Ozone is Redline Facilitation Group.
+Ozone is an Operational Executor Agent.
 
-##### A.6.1.2.2.2 - Operational GovOps [Core]  <!-- UUID: a491d7d0-3e0b-4b6f-a9a7-25b19b6f7117 -->
+##### A.6.1.2.2.2 - Sky Primitives [Core]  <!-- UUID: a491d7d0-3e0b-4b6f-a9a7-25b19b6f7117 -->
 
-Operational GovOps for Operational Executor Agent Ozone is Soter Labs.
+The documents herein implement the Sky Primitives for Ozone See [A.2.2 - Sky Primitives](https://sky-atlas.io/#fcde2604-a138-4c1b-9d9a-14895835c907).
+
+###### A.6.1.2.2.2.1 - Genesis Primitives [Core]  <!-- UUID: 3b8f9996-bef3-4147-940d-da169fd78822 -->
+
+The documents herein implement the Genesis Primitives for Ozone. See [A.2.2.4 - Genesis Primitives](https://sky-atlas.io/#3d5e3668-8333-4908-adcc-5784cfe7f6b5).
+
+###### A.6.1.2.2.2.1.1 - Agent Creation Primitive [Core]  <!-- UUID: e9ca5d8f-57b9-48f0-b6bd-75f7916ed5fe -->
+
+The documents herein contain all data and specifications for Ozone's Instance of the Agent Creation Primitive. [See A.2.2.4.1 - Agent Creation Primitive](https://sky-atlas.io/#82b95f6d-4883-4f08-ac3a-9d8189013fbe).
+
+###### A.6.1.2.2.2.1.1.1 - Primitive Hub Document [Core]  <!-- UUID: 61789041-82e5-45ff-a608-341c4704c50f -->
+
+The documents herein organize all base information relevant to Ozone's usage of the Agent Creation Primitive.
+
+###### A.6.1.2.2.2.1.1.1.1 - Global Activation Status [Core]  <!-- UUID: 62ef4c8b-500a-4f73-a719-1ecc5a3f70a2 -->
+
+`Completed`
+
+###### A.6.1.2.2.2.1.1.1.2 - Active Instances Directory - [Core]  <!-- UUID: fdac55c0-5ba9-470d-9ab0-180022554243 -->
+    
+This document contains a Directory of all Instances of the Agent Creation Primitive with Instance status of `Active`.
+
+###### A.6.1.2.2.2.1.1.1.3 - Completed Instances Directory - [Core]  <!-- UUID: 29ea7b0d-6726-4aa2-bcfd-08cb2fb22726 -->
+
+This document contains a Directory of all Instances of the Agent Creation Primitive with Instance status of `Completed`.
+
+###### A.6.1.2.2.2.1.1.1.3.1 - Single Instance Configuration Document Location - [Core]  <!-- UUID: a86cde6f-741b-4bfc-9bc5-d6da7320ec0a -->
+
+This Instance's associated Instance Configuration Document is located at A.6.1.2.2.2.1.1.3.1(https://sky-atlas.io/#6cdce0ea-4489-4e97-8688-d5bee008da8d)
+
+###### A.6.1.2.2.2.1.1.1.4 - In Progress Invocations Directory - [Core]  <!-- UUID: fb329403-adbd-4321-bd14-061485993d26 -->
+
+Because the Agent Creation Primitive is deployed solely for the one-time creation of the Agent, no further Instances of the Primitive can be Invoked.
+
+###### A.6.1.2.2.2.1.1.1.5 - Hub Data Repository - [Core]  <!-- UUID: c86d319b-3e87-443e-abba-7e2322f986cf -->
+
+The documents herein contains the Data Repository for the Primitive Hub Document.
+
+###### A.6.1.2.2.2.1.1.1.5.1 - Archived Invocations/Instances - [Core]  <!-- UUID: aab98bf6-5367-4dac-a6b0-a4a7cefb2cd0 -->
+
+The subtrees for archived Invocations and Instances of the Agent Creation Primitive are stored here.
+
+###### A.6.1.2.2.2.1.1.1.5.1.1 - Failed Invocations - [Core]  <!-- UUID: 5a685878-efb5-44c6-9b2b-dd49e96c13ae -->
+
+The subtrees for failed Invocations of the Agent Creation Primitive are stored here.
+
+###### A.6.1.2.2.2.1.1.1.5.1.2 - Suspended Instances - [Core]  <!-- UUID: c1279ebb-53b2-426f-9cf8-2664c954e577 -->
+
+The subtrees for Instances of the Agent Creation Primitive with `Suspended` Status are stored here.
+
+###### A.6.1.2.2.2.1.1.2 - Active Instances [Core]  <!-- UUID: 213559bb-a494-438e-beb6-6239b50459e4 -->
+
+The Instances of the Agent Creation Primitive with `Active` Status are stored herein.
+
+###### A.6.1.2.2.2.1.1.3 - Completed Instances [Core]  <!-- UUID: 48a3feef-5c28-401c-906c-1718eee93923 -->
+
+The Instances of the Agent Creation Primitive with `Completed` Status are contained herein.
+
+###### A.6.1.2.2.2.1.1.3.1 - Single Instance Configuration Document Location - [Core]  <!-- UUID: 6cdce0ea-4489-4e97-8688-d5bee008da8d -->
+
+The documents herein contain the Instance Configuration Document for the Single Agent Creation Primitive Instance.
+
+###### A.6.1.2.2.2.1.1.3.1.1 - Parameters - [Core]  <!-- UUID: 5d6cff9f-e7b6-46f9-8d05-fff27058f024 -->
+
+The documents herein define the parameters of the Single Instance of the Agent Creation Primitive.
+
+###### A.6.1.2.2.2.1.1.3.1.1.1 - Name - [Core]  <!-- UUID: 1a6aa46d-b91d-4100-831b-3f998d52c83d -->
+
+The name of the Agent is Ozone.
+
+###### A.6.1.2.2.2.1.1.3.1.1.2 - SubProxy Account - [Core]  <!-- UUID: d9ad95b5-c534-4d23-8a9d-cfbb39f14862 -->
+
+The address of Ozone's SubProxy Account on the Ethereum Mainnet is `0x9FE628BFc33f0352Bb1f93168881a9Ef93C8d2CF`.
+
+###### A.6.1.2.2.2.1.1.3.1.1.3 - StarGuard Contract - [Core]  <!-- UUID: 045d4484-ecf0-4e1f-8c17-d8f2eccf8771 -->
+
+The address of Ozone's StarGuard contract on the Ethereum Mainnet is '0x9803DA8a51Fa02EEbEc3B1b969a9B80f9115cD80'.
+
+###### A.6.1.2.2.2.1.1.3.1.1.3.1 - StarGuard Max Delay - [Core]  <!-- UUID: eb48cc13-b40c-4c2b-a3eb-f3c06ea1441a -->
+
+The Ozone StarGuard `maxDelay` is seven (7) days.
+
+###### A.6.1.2.2.2.1.1.3.1.1.4 - Genesis Account - [Core]  <!-- UUID: de2da5e3-7b2a-400a-b1d3-42d4bc78b41e -->
+
+The address of Ozone's Genesis Account will be specified in a future iteration of the Ozone Artifact.
+
+###### A.6.1.2.2.2.1.1.3.1.1.5 - Custom Instance Parameters - [Core]  <!-- UUID: 3606f289-e241-4c52-ba07-cb5272f67754 -->
+
+The documents herein define the custom parameters of the Single Instance of the Agent Creation Primitive, if any.
+
+###### A.6.1.2.2.2.1.1.3.1.1.5.1 - Redline - [Core]  <!-- UUID: c7ea4f3c-d427-4e0c-9f3f-b1ae0c0e3347 -->
+
+The Operational Facilitator for Operational Executor Agent Ozone is Redline Facilitation Group. Redline Facilitation Group interprets the Atlas and Artifacts on behalf of the Agents.
+
+###### A.6.1.2.2.2.1.1.3.1.1.5.2 - Soter Labs - [Core]  <!-- UUID: 384d29b0-8621-4dfa-a6c3-23b1397a9712 -->
+
+Operational GovOps for Operational Executor Agent Ozone is Soter Labs. Soter Labs plays a crucial role in implementing Prime Agent strategies, doing so by executing the technical specifications outlined in Primitive Configuration Documents of Agent Artifacts.
+
+###### A.6.1.2.2.2.1.1.3.1.2 - Operational Process Definition - [Core]  <!-- UUID: f44a3ad0-7cc0-4756-8e00-3846092da92a -->
+
+Because the Agent Creation Primitive is deployed solely for the one-time creation of the Agent, no further operational process is needed post-deployment.
+
+###### A.6.1.2.2.2.1.1.3.1.3 - Data Repository - [Core]  <!-- UUID: a13c375b-9a89-41d5-bff7-9559cfae2c86 -->
+
+The documents herein contain data relevant to the Single Instance of the Agent Creation Primitive.
+
+###### A.6.1.2.2.2.1.1.3.1.3.1 - Initial Planning - [Core]  <!-- UUID: 5b4bb11d-09da-4ac0-938a-99b514f0810e -->
+
+The materials associated with initial planning of the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.1.3.1.3.2 - Operational GovOps Review - [Core]  <!-- UUID: e9d88d0e-9813-4e98-82c2-6197fb2ebbba -->
+
+The materials associated with Operational GovOps Review during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.1.3.1.3.3 - Artifact Edit Proposal - [Core]  <!-- UUID: b6ebdecd-8317-4a1e-9b71-1a0347004c70 -->
+
+The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.2 - Prime Transformation Primitive [Core]  <!-- UUID: b09826f5-7751-4c64-a79c-ebf72840bb43 -->
+
+The documents herein contain all data and specifications for Ozone's instance of the Prime Transformation Primitive. See [A.2.2.4.2 - Prime Transformation Primitive](https://sky-atlas.io/#81411106-fd6d-4f9c-b3ae-7af7b5e62482).
+
+###### A.6.1.2.2.2.1.2.1 - Primitive Hub Document [Core]  <!-- UUID: 1dc0ba03-55d6-4748-a6b4-32f8c502b813 -->
+
+The documents herein organize all base information relevant to Ozone's usage of the Prime Transformation Primitive.
+
+###### A.6.1.2.2.2.1.2.1.1 - Global Activation Status [Core]  <!-- UUID: 8fe37627-8f0a-4e0d-9073-203901bf2050 -->
+
+`Inactive`
+
+###### A.6.1.2.2.2.1.2.1.2 - Active Instances Directory [Core]  <!-- UUID: 9984aada-40f0-41cf-811c-2e7ad1174401 -->
+
+This document contains a Directory of all Instances of the Prime Transformation Primitive with Instance status of `Active`.
+
+###### A.6.1.2.2.2.1.2.1.3 - Completed Instances Directory [Core]  <!-- UUID: ab59b68f-55c9-4655-a8a5-d66c2a94812e -->
+
+This document contains a Directory of all Instances of the Prime Transformation Primitive with Instance status of `Completed`.
+
+###### A.6.1.2.2.2.1.2.1.4 - In Progress Invocations Directory [Core]  <!-- UUID: 4a35b179-a640-4fa0-9a8e-dfb0b40046ea -->
+
+Because the Prime Transformation Primitive is deployed solely for the one-time transformation of the Agent, no further Instances of the Primitive can be Invoked.
+
+###### A.6.1.2.2.2.1.2.1.5 - Hub Data Repository [Core]  <!-- UUID: ae50734a-3816-47d3-9572-f4ceb261b6b0 -->
+
+The documents herein contain the Data Repository for the Primitive Hub Document.
+
+###### A.6.1.2.2.2.1.2.1.5.1 - Archived Invocations/Instances [Core]  <!-- UUID: 7860b3fb-abda-4df9-aade-d4c851b25ed6 -->
+
+The subtrees for archived Invocations and Instances of the Prime Transformation Primitive are stored here.
+
+###### A.6.1.2.2.2.1.2.1.5.1.1 - Failed Invocations [Core]  <!-- UUID: 9cd01fe4-0498-45ec-9d26-6bf35025e850 -->
+
+The subtrees for failed Invocations of the Prime Transformation Primitive are stored here.
+
+###### A.6.1.2.2.2.1.2.1.5.1.2 - Suspended Instances [Core]  <!-- UUID: f34c09b7-3897-4d4a-a160-bde1a008dd34 -->
+
+The subtrees for Instances of the Prime Transformation Primitive with `Suspended` Status are stored here.
+
+###### A.6.1.2.2.2.1.2.2 - Active Instances [Core]  <!-- UUID: 3b9f0bdb-cc86-4402-88d9-304ec0f3752e -->
+
+The Instances of the Prime Transformation Primitive with `Active` Status are stored herein.
+
+###### A.6.1.2.2.2.1.2.3 - Completed Instances [Core]  <!-- UUID: ea1fc101-2d2d-449f-9933-d770d265b7db -->
+
+The Instances of the Prime Transformation Primitive with `Completed` Status are contained herein.
+
+###### A.6.1.2.2.2.1.2.4 - In Progress Invocations [Core]  <!-- UUID: 1538e819-f3a8-4ed2-a32c-5a500094bb36 -->
+
+Because the Prime Transformation Primitive is deployed solely for the one-time transformation of the Agent, no further Instances of the Primitive can be Invoked.
+
+###### A.6.1.2.2.2.1.3 - Executor Transformation Primitive [Core]  <!-- UUID: 680e85a2-1d72-4b8f-88b7-f83517a5ad1a -->
+
+The documents herein contain all data and specifications for Ozone's Instance of the Executor Transformation Primitive. See [A.2.2.4.3 - Executor Transformation Primitive](https://sky-atlas.io/#2f249be5-8edb-41e4-b429-734e1ba2cbc7).
+
+###### A.6.1.2.2.2.1.3.1 - Primitive Hub Document [Core]  <!-- UUID: 14545d28-d46e-4355-a4c0-340f84093f82 -->
+
+The documents herein organize all base information relevant to Ozone's usage of the Executor Transformation Primitive.
+
+###### A.6.1.2.2.2.1.3.1.1 - Global Activation Status [Core]  <!-- UUID: 93d44cc6-caa3-4f47-9595-e9aa57125e11 -->
+
+`Completed`
+
+###### A.6.1.2.2.2.1.3.1.2 - Active Instances Directory [Core]  <!-- UUID: 2896c62e-6c7d-4479-9c27-d20b69e4149d -->
+
+This document contains a Directory of all Instances of the Executor Transformation Primitive with Instance status of `Active`.
+
+###### A.6.1.2.2.2.1.3.1.3 - Completed Instances Directory [Core]  <!-- UUID: 0367dfaa-79ff-4e77-81ec-5d792583cb86 -->
+
+This document contains a Directory of all Instances of the Executor Transformation Primitive with Instance status of `Completed`.
+
+###### A.6.1.2.2.2.1.3.1.3.1 - Single Instance Configuration Document Location [Core]  <!-- UUID: 7dac1055-e4a1-4031-8d09-6fbedcc074f2 -->
+
+This Instance’s associated Instance Configuration Document is located at A.6.1.2.2.X.X.X.X.X (https://sky-atlas.io/#f5632ed7-a73b-4332-b53b-78a153c3fd0d)
+
+###### A.6.1.2.2.2.1.3.1.4 - In Progress Invocations Directory [Core]  <!-- UUID: 5eb2502f-2da0-41ec-89b7-2a35b52fc7c8 -->
+
+Because the Executor Transformation Primitive is deployed solely for the one-time transformation of the Agent, no further Instances of the Primitive can be Invoked.
+
+###### A.6.1.2.2.2.1.3.1.5 - Hub Data Repository [Core]  <!-- UUID: a3cc6280-ee66-46dc-9045-97006db363f5 -->
+
+The documents herein contain the Data Repository for the Primitive Hub Document.
+
+###### A.6.1.2.2.2.1.3.1.5.1 - Archived Invocations/Instances [Core]  <!-- UUID: d8f7a5cb-4029-4eac-9d41-98c631f9d3ca -->
+
+The subtrees for archived Invocations and Instances of the Executor Transformation Primitive are stored here.
+
+###### A.6.1.2.2.2.1.3.1.5.1.1 - Failed Invocations [Core]  <!-- UUID: 7b60b6eb-598b-4c07-bc61-f5373500cc80 -->
+
+The subtrees for failed Invocations of the Executor Transformation Primitive are stored here.
+
+###### A.6.1.2.2.2.1.3.1.5.1.2 - Suspended Instances [Core]  <!-- UUID: 60ce768e-be51-44ee-a2e4-9a2272ad29e4 -->
+
+The subtrees for Instances of the Executor Transformation Primitive with `Suspended` Status are stored here.
+
+###### A.6.1.2.2.2.1.3.2 - Actives Instances [Core]  <!-- UUID: 874b5c21-bce9-42fe-aad4-68bf721ee03a -->
+
+The Instances of the Executor Transformation Primitive with `Active` Status are stored herein.
+
+###### A.6.1.2.2.2.1.3.3 - Completed Instances [Core]  <!-- UUID: 5bb08d1b-1834-4e15-b704-f48f1472daa8 -->
+
+The Instances of the Executor Transformation Primitive with `Completed` Status are stored herein.
+
+###### A.6.1.2.2.2.1.3.3.1 - Single Instance Configuration Document [Core]  <!-- UUID: f5632ed7-a73b-4332-b53b-78a153c3fd0d -->
+
+The documents herein contain the Instance Configuration Document for the Single Executor Transformation Primitive Instance.
+
+###### A.6.1.2.2.2.1.3.3.1.1 - Parameters [Core]  <!-- UUID: a0baad41-d075-4031-a9a4-77f9b6ef6cd4 -->
+
+The documents herein define the parameters of the Ozone Instance of the Executor Transformation Primitive.
+
+###### A.6.1.2.2.2.1.3.3.1.1.1 - Agent Type [Core]  <!-- UUID: 6877f189-a7e8-4962-ac37-fa341b3485ef -->
+
+Ozone is an Executor Agent.
+
+###### A.6.1.2.2.2.1.3.3.1.1.2 - Custom Instance Parameters [Core]  <!-- UUID: fbb01569-0b60-4771-a9e1-2fcc29050c73 -->
+
+The documents herein define the custom parameters of the Single Instance of the Executor Transformation Primitive, if any.
+
+###### A.6.1.2.2.2.1.3.3.1.2 - Operational Process Definition [Core]  <!-- UUID: c77499fa-8221-449b-a3ca-e0fafc0ac668 -->
+
+Because the Executor Transformation Primitive is deployed solely for the one-time transformation of the Agent, no further operational process is needed post-deployment.
+
+###### A.6.1.2.2.2.1.3.3.1.3 - Data Repository [Core]  <!-- UUID: 43d61feb-94f4-4997-a5de-c5ed6b4925b6 -->
+
+The documents herein contain data relevant to the Single Instance of the Executor Transformation Primitive.
+
+###### A.6.1.2.2.2.1.3.3.1.3.1 - Initial Planning [Core]  <!-- UUID: e59be66d-1646-4dd3-979c-4a01bf9b07f9 -->
+
+The materials associated with initial planning of the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.3.3.1.3.2 - Operational GovOps Review [Core]  <!-- UUID: 747823c4-316d-4d2d-b9d2-c2eaeafc7222 -->
+
+The materials associated with Operational GovOps Review during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.3.3.1.3.3 - Artifact Edit Proposal [Core]  <!-- UUID: be658dce-7f22-4338-89ae-fdda54984f15 -->
+
+The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.3.4 - In Progress Invocations [Core]  <!-- UUID: f50752d6-999a-447f-ae8b-61ae113372fe -->
+
+Because the Executor Transformation Primitive is deployed solely for the one-time transformation of the Agent, no further Instances of the Primitive can be Invoked.
+
+###### A.6.1.2.2.2.1.4 - Agent Token Primitive [Core]  <!-- UUID: eddf99e2-4f1f-4906-a378-0179c40a09a9 -->
+
+The documents herein contain all data and specifications for Ozone's Instance of the Agent Token Primitive. See [A.2.2.4.4 - Agent Token Primitive](https://sky-atlas.io/#2047c361-db28-4952-a70c-83d07b562064).
+
+###### A.6.1.2.2.2.1.4.1 - Primitive Hub Document [Core]  <!-- UUID: 1b551ec1-95a8-401b-9c2e-a234b78260d4 -->
+
+The documents herein organize all base information relevant to Ozone's usage of the Agent Token Primitive.
+
+###### A.6.1.2.2.2.1.4.1.1 - Global Activation Status [Core]  <!-- UUID: 15ce4b3c-482a-4a7f-b7d5-2967c684cd79 -->
+
+`Active`
+
+###### A.6.1.2.2.2.1.4.1.2 - Active Instances Directory [Core]  <!-- UUID: 8cf7bb89-0e06-462d-8b19-18e599017281 -->
+
+This document contains a Directory of all Instances of the Agent Token Primitive with Instance status of `Active`.
+
+###### A.6.1.2.2.2.1.4.1.2.1 - Single Instance Configuration Document Location [Core]  <!-- UUID: 482315a0-eebd-4f54-bf1f-b3e97804d27c -->
+
+This Instance’s associated Instance Configuration Document is located at A.6.1.2.2.X.X.X.X.X (https://sky-atlas.io/#b14f7788-4a35-4d31-a0f2-b4add513556a)
+
+###### A.6.1.2.2.2.1.4.1.3 - Completed Instances Directory [Core]  <!-- UUID: 693b22cf-7a47-4b35-bd0d-ec794510042b -->
+
+This document contains a Directory of all Instances of the Agent Token Primitive with Instance status of `Completed`.
+
+###### A.6.1.2.2.2.1.4.1.4 - In Progress Invocations Directory [Core]  <!-- UUID: b20f58de-1787-4f0a-97de-8e71377dcc4b -->
+
+Because the Agent Token Primitive is Invoked solely for the one-time deployment of the Agent’s token, no further Instances of the Primitive can be Invoked.
+
+###### A.6.1.2.2.2.1.4.1.5 - Hub Data Repository [Core]  <!-- UUID: 9d5e0292-72e0-46be-84eb-1d0fc7cbfec6 -->
+
+The documents herein contains the Data Repository for the Primitive Hub Document.
+
+###### A.6.1.2.2.2.1.4.1.5.1 - Archived Invocations/Instances [Core]  <!-- UUID: f7d4a59b-64b4-4dc1-8671-6ccb5df2e747 -->
+
+The subtrees for archived Invocations and Instances of the Agent Token Primitive are stored here.
+
+###### A.6.1.2.2.2.1.4.1.5.1.1 - Failed Invocations [Core]  <!-- UUID: acef4f74-a1c4-445e-b7e3-5d46582e3658 -->
+
+The subtrees for failed Invocations of the Agent Token Primitive are stored here.
+
+###### A.6.1.2.2.2.1.4.1.5.1.2 - Suspended Instances [Core]  <!-- UUID: 25c18e9c-01e1-41c6-9d2e-15bf98e80d8b -->
+
+The subtrees for Instances of the Agent Token Primitive with `Suspended` Status are stored here.
+
+###### A.6.1.2.2.2.1.4.2 - Active Instances [Core]  <!-- UUID: f2e36ae9-5d05-449f-a7ac-095b49b0fc7d -->
+
+The Instances of the Agent Token Primitive with `Active` Status are stored herein.
+
+###### A.6.1.2.2.2.1.4.2.1 - Single Instance Configuration Document [Core]  <!-- UUID: b14f7788-4a35-4d31-a0f2-b4add513556a -->
+
+The documents herein contain the Instance Configuration Document for the Single Agent Token Primitive Instance.
+
+###### A.6.1.2.2.2.1.4.2.1.1 - Parameters [Core]  <!-- UUID: efc502e6-138e-49b5-986a-9349ced44c30 -->
+
+The documents herein define the parameters of the Single Instance of the Agent Token Primitive.
+
+###### A.6.1.2.2.2.1.4.2.1.1.1 - Token Name [Core]  <!-- UUID: f1ec92d7-61b7-4d06-9c8a-c4d14cee3bcc -->
+
+The name of Ozone's token is Ozone.
+
+###### A.6.1.2.2.2.1.4.2.1.1.2 - Token Symbol [Core]  <!-- UUID: ab6beaa8-055b-4f47-a6c1-0d89b0ee05b4 -->
+
+The symbol of Ozone's token is OZONE.
+
+###### A.6.1.2.2.2.1.4.2.1.1.3 - Genesis Supply [Core]  <!-- UUID: eb97e515-ca94-4ab8-9e30-17815b2924d4 -->
+
+The Genesis Supply of OZONE will be specified in a future iteration of the Ozone Artifact.
+
+###### A.6.1.2.2.2.1.4.2.1.1.4 - Token Address [Core]  <!-- UUID: f0dd762e-5292-45b2-9bea-faa50e025441 -->
+
+The address of OZONE will be specified in a future iteration of the Ozone Artifact.
+
+###### A.6.1.2.2.2.1.4.2.1.1.5 - Token Admin [Core]  <!-- UUID: db9986f0-ebfa-41fb-b179-06b63df260ee -->
+
+The token Admin will be specified in a future iteration of the Ozone Artifact.
+
+###### A.6.1.2.2.2.1.4.2.1.1.6 - Token Emissions [Core]  <!-- UUID: cf671d69-692d-4402-8d8b-d9cac057dcba -->
+
+Token emissions beyond the Genesis Supply are permanently disabled; this cannot be reverted by Ozone Governance. Sky Governance retains the ability to revert where Ozone is in violation of Risk Capital requirements and emissions are required by the Risk Framework. See [A.3.2 - Risk Capital](https://sky-atlas.io/#55999acf-75fe-4adf-8584-9746ef50d3e4).
+
+###### A.6.1.2.2.2.1.4.2.1.1.7 - Custom Instance Parameters [Core]  <!-- UUID: 0204b91e-a516-4567-b292-6eea4ee68957 -->
+
+The documents herein define the custom parameters of the Single Instance of the Agent Token Primitive, if any.
+
+###### A.6.1.2.2.2.1.4.2.1.2 - Operational Process Definition [Core]  <!-- UUID: 8444d887-0744-4991-847c-a06461498467 -->
+
+The documents herein define the operational processes for minting and initial distribution of the tokens from the Genesis Supply.
+
+- These processes will be defined in a future iteration of the Ozone Artifact.
+
+###### A.6.1.2.2.2.1.4.2.1.3 - Data Repository [Core]  <!-- UUID: bee0cecd-17da-4af3-9419-22a61ae04c3a -->
+
+The documents herein contain data relevant to the Single Instance of the Agent Token Primitive.
+
+###### A.6.1.2.2.2.1.4.2.1.3.1 - Initial Planning [Core]  <!-- UUID: 10179d38-3092-4aa5-bfe3-84b93f78fbde -->
+
+The materials associated with initial planning of the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.4.2.1.3.2 - Operational GovOps Review [Core]  <!-- UUID: 0ab3c3d4-e064-4625-b423-18291131bd6d -->
+
+The materials associated with Operational GovOps Review during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.4.2.1.3.3 - Artifact Edit Proposal [Core]  <!-- UUID: a65e5fba-2d37-402c-9389-3366343da2e7 -->
+
+The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.1.4.3 - Completed Instances [Core]  <!-- UUID: ef6da6df-21de-40ae-b41e-fdbc1ec9783f -->
+
+The Instances of the Agent Token Primitive with `Completed` Status are stored herein.
+
+###### A.6.1.2.2.2.1.4.4 - In Progress Invocations [Core]  <!-- UUID: 415592d7-9c78-430e-89ea-22f61feaf604 -->
+
+Because the Agent Token Primitive is Invoked solely for the one-time deployment of the Agent’s token, no further Instances of the Primitive can be Invoked.
+
+###### A.6.1.2.2.2.2 - Operational Primitives [Core]  <!-- UUID: 369801f5-ee6d-4154-980e-f2fe53da56e8 -->
+
+The documents herein implement the Operational Primitives for Ozone. See [A.2.2.5 - Operational Primitives](https://sky-atlas.io/#0192ec95-9207-480e-8c51-88d2a1da95ad).
+
+###### A.6.1.2.2.2.2.1. - In Progress Invocations [Core]  <!-- UUID: 57be2e96-d750-4181-8f19-82d5c2e9a3d3 -->
+
+The in progress Invocations of the Executor Accord Primitive are contained herein. Once an Invocation is successfully completed, its subtree will be moved to A.6.1.2.2.2.2.1.2 - Active Instances.(https://sky-atlas.io/#b467f6aa-1e79-445a-820f-e101c08dcf0c) 
+
+###### A.6.1.2.2.2.2.1.1 - Primitive Hub Document [Core]  <!-- UUID: 7e33110a-7a64-4908-9fec-310334dd88d3 -->
+
+The documents herein organize all base information relevant to Ozone's usage of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.1.1 - Global Activation Status [Core]  <!-- UUID: 81f55528-50fb-4551-a1d3-5cbf89d1a11d -->
+
+`Active`
+
+###### A.6.1.2.2.2.2.1.1.2 - Active Instances Directory [Core]  <!-- UUID: 5dd90b01-9b2e-4cf7-95ef-e0a2dc0307d0 -->
+
+This document contains a Directory of all Instances of the Executor Accord Primitive with Instance status of `Active`.
+
+###### A.6.1.2.2.2.2.1.1.2.1 - Skybase-Ozone Instance Configuration Document Location [Core]  <!-- UUID: 09f629e7-8619-4ec1-9f45-800e313269d6 -->
+
+This Instance’s associated Instance Configuration Document is located at A.6.1.2.2.2.2.1.2.1 (https://sky-atlas.io/#95f75157-5eb7-4740-a7d2-2f0bebcf3e18)
+
+###### A.6.1.2.2.2.2.1.1.2.2 - Obex-Ozone Instance Configuration Document Location [Core]  <!-- UUID: ce874980-2be5-4298-9f67-335e2bbdabac -->
+
+This Instance’s associated Instance Configuration Document is located at A.6.1.2.2.2.2.1.2.2 (https://sky-atlas.io/#4ba72ac4-81f7-4083-a844-4f23b5e3b1fd)
+
+###### A.6.1.2.2.2.2.1.1.2.3 - Pattern-Ozone Instance Configuration Document Location [Core]  <!-- UUID: db65802e-4647-445f-b824-5801687365da -->
+
+This Instance’s associated Instance Configuration Document is located at A.6.1.2.2.2.2.1.2.3 (https://sky-atlas.io/#ce86f57a-3217-48ec-88cf-bae310234105)
+
+###### A.6.1.2.2.2.2.1.1.2.4 - Osero-Ozone Instance Configuration Document Location [Core]  <!-- UUID: 1d34732a-5134-4919-afc0-aacf51e13d46 -->
+
+This Instance’s associated Instance Configuration Document is located at A.6.1.2.2.2.2.1.2.4 (https://sky-atlas.io/#dd0515f7-64fd-411e-8dff-07fc1b2be962)
+
+###### A.6.1.2.2.2.2.1.1.2.5 - Launch Agent 7-Ozone Instance Configuration Document Location [Core]  <!-- UUID: c8e75b0f-6a3b-45fe-9c7e-e287a81686a8 -->
+
+This Instance’s associated Instance Configuration Document is located at A.6.1.2.2.2.2.1.2.5 (https://sky-atlas.io/#6843c012-4dd3-4255-bb7b-667413496680)
+
+###### A.6.1.2.2.2.2.1.1.3 - Completed Instances Directory [Core]  <!-- UUID: bf5d7c00-3386-44c8-8608-854ec9b831db -->
+
+This document contains a Directory of all Instances of the Executor Accord Primitive with Instance status of `Completed`.
+
+###### A.6.1.2.2.2.2.1.1.4 - In Progress Invocations Directory [Core]  <!-- UUID: c6f2ae43-0751-4f89-bd72-fec6f3ea7a78 -->
+
+This document contains a Directory of all prospective Instances of the Executor Accord Primitive whose Invocation is currently in progress. Invocations that are completed successfully are moved to [A.6.1.2.2.2.2.1.1.2 - Active Instances Directory](https://sky-atlas.io/#5dd90b01-9b2e-4cf7-95ef-e0a2dc0307d), whereas failed Invocations are Archived in [A.6.1.2.2.2.2.1.1.5 - Hub Data Repository](https://sky-atlas.io/#4c4dd0fc-66b7-422b-8445-2d6ed13e31f7).
+
+###### A.6.1.2.2.2.2.1.1.5 - Hub Data Repository [Core]  <!-- UUID: 4c4dd0fc-66b7-422b-8445-2d6ed13e31f7 -->
+
+The documents herein contains the Data Repository for the Primitive Hub Document.
+
+###### A.6.1.2.2.2.2.1.1.5.1 Archived Invocations/Instances [Core]  <!-- UUID: 01c4257b-2a6a-4505-8f53-5bcbbc3b29a1 -->
+
+The subtrees for archived Invocations and Instances of the Executor Accord Primitive are stored here.
+
+###### A.6.1.2.2.2.2.1.1.5.1.1 Failed Invocations [Core]  <!-- UUID: a1021250-36a4-461f-9769-79fd916286f2 -->
+
+The subtrees for failed Invocations of the Executor Accord Primitive are stored here.
+
+###### A.6.1.2.2.2.2.1.1.5.1.2 Suspended Instances [Core]  <!-- UUID: eb4ab435-070c-41b8-91bc-8ee95ccd351f -->
+
+The subtrees for Instances of the Executor Accord Primitive with `Suspended` Status are stored here. 
+
+###### A.6.1.2.2.2.2.1.2 - Active Instances [Core]  <!-- UUID: b467f6aa-1e79-445a-820f-e101c08dcf0c -->
+
+The Instances of the Executor Accord Primitive with `Active` Status are stored herein.
+
+###### A.6.1.2.2.2.2.1.2.1 - Skybase-Ozone Instance Configuration Document [Core]  <!-- UUID: 95f75157-5eb7-4740-a7d2-2f0bebcf3e18 -->
+
+The documents herein contain the Instance Configuration Document for the Skybase-Ozone Executor Accord Primitive Instance.
+
+###### A.6.1.2.2.2.2.1.2.1.1 - Parameters [Core]  <!-- UUID: 94734aac-7b37-4d26-a787-82c8962f3398 -->
+
+The documents herein define the parameters of the Skybase-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.1.1.1 - Prime Agent [Core]  <!-- UUID: 350cffeb-4ba0-4365-95a9-7feb74a147ee -->
+
+The Skybase Prime Agent details, including foundation and development company are specified in [A.2.8.2.7.1.1.2 - Skybase Prime Details](https://sky-atlas.io/#287bbf07-b29b-46b1-a8a1-d237825424cd).
+
+###### A.6.1.2.2.2.2.1.2.1.1.2 - Custom Instance Parameters [Core]  <!-- UUID: 723fedae-9b82-412e-91e9-50be1514a0a7 -->
+
+The documents herein define the custom parameters of the Skybase-Ozone Instance of the Executor Accord Primitive, if any.
+
+###### A.6.1.2.2.2.2.1.2.1.2 - Operational Process Definition [Core]  <!-- UUID: f6ea2149-abf8-49fe-b05e-ee775956dcb8 -->
+
+The documents herein define the process for the ongoing management of the Skybase-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.1.3 - Data Repository [Core]  <!-- UUID: 523c023c-315b-447d-b28f-a6194a6a069f -->
+
+The documents herein contain data relevant to the Skybase-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.1.3.1 - Initial Planning [Core]  <!-- UUID: e3da9d58-485d-4d6a-a1ab-7c5404b79128 -->
+
+The materials associated with initial planning of the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.1.3.2 - Operational GovOps Review [Core]  <!-- UUID: ed203ada-877f-4b4d-a9c2-c3f2ec6da695 -->
+
+The materials associated with Operational GovOps Review during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.1.3.3 - Artifact Edit Proposal [Core]  <!-- UUID: b86344bd-6c65-4838-b046-8ca5bdd61da1 -->
+
+The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.2 - Obex-Ozone Instance Configuration Document [Core]  <!-- UUID: 4ba72ac4-81f7-4083-a844-4f23b5e3b1fd -->
+
+The documents herein contain the Instance Configuration Document for the Obex-Ozone Executor Accord Primitive Instance.
+
+###### A.6.1.2.2.2.2.1.2.2.1 - Parameters [Core]  <!-- UUID: 9e94d7d4-578d-4671-8c66-b107f9528dfd -->
+
+The documents herein define the parameters of the Obex-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.2.1.1 - Prime Agent [Core]  <!-- UUID: a1a08c72-632d-4983-a769-1ecac2146075 -->
+
+The Obex Prime Agent details, including foundation and development company are specified in [A.2.8.2.4.1.1.2 - Obex Details](https://sky-atlas.io/#665a712a-d211-4a7a-b4c9-a8bad61b3f9c).
+
+###### A.6.1.2.2.2.2.1.2.2.1.2 - Custom Instance Parameters [Core]  <!-- UUID: 65f407cb-597a-48a5-a84a-e88250acfc59 -->
+
+The documents herein define the custom parameters of the Obex-Ozone Instance of the Executor Accord Primitive, if any.
+
+###### A.6.1.2.2.2.2.1.2.2.2 - Operational Process Definition [Core]  <!-- UUID: c5c0ec50-020b-4b1e-8787-982334f8ab1b -->
+
+The documents herein define the process for the ongoing management of the Obex-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.2.3 - Data Repository [Core]  <!-- UUID: 77e9ccdf-2941-47ed-9bca-4fd4712a9afe -->
+
+The documents herein contain data relevant to the Obex-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.2.3.1 - Initial Planning [Core]  <!-- UUID: d359bb46-b925-4913-b941-ffbaf2b4679e -->
+
+The materials associated with initial planning of the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.2.3.2 - Operational GovOps Review [Core]  <!-- UUID: 5b72cec1-469c-4f72-8266-82ccb4547223 -->
+
+The materials associated with Operational GovOps Review during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.2.3.3 - Artifact Edit Proposal [Core]  <!-- UUID: 04c1fbe5-61c2-445d-8b6c-612a4ff3c23b -->
+
+The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.3 - Pattern-Ozone Instance Configuration Document [Core]  <!-- UUID: ce86f57a-3217-48ec-88cf-bae310234105 -->
+
+The documents herein contain the Instance Configuration Document for the Pattern-Ozone Executor Accord Primitive Instance.
+
+###### A.6.1.2.2.2.2.1.2.3.1 - Parameters [Core]  <!-- UUID: b334ee30-33c1-4d91-b067-65c6923b43f8 -->
+
+The documents herein define the parameters of the Pattern-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.3.1.1 - Prime Agent [Core]  <!-- UUID: 7f3bf992-1dcc-47ce-a6c7-bca5aa157b89 -->
+
+The Pattern Prime Agent’s development company is specified in [A.6.1.1.6.2.1.1.3.1.1.5.1 - Pattern Dev Co.](https://sky-atlas.io/#07037190-1bad-41df-9fa3-012c8cd18bdf) Other Pattern Prime Agent details will be specified in future iterations of the Atlas.
+
+###### A.6.1.2.2.2.2.1.2.3.1.2 - Custom Instance Parameters [Core]  <!-- UUID: 71305335-1f72-4d84-9677-e22cf20a1ea1 -->
+
+The documents herein define the custom parameters of the Pattern-Ozone Instance of the Executor Accord Primitive, if any.
+
+###### A.6.1.2.2.2.2.1.2.3.2 - Operational Process Definition [Core]  <!-- UUID: f1575f5b-9490-4649-bc7c-98ad396122d0 -->
+
+The documents herein define the process for the ongoing management of the Pattern-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.3.3 - Data Repository [Core]  <!-- UUID: e1fee1f2-09d8-41bb-a179-4927997ab408 -->
+
+The documents herein contain data relevant to the Pattern-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.3.3.1 - Initial Planning [Core]  <!-- UUID: e1f74240-3d71-4404-8484-2fd8d9275678 -->
+
+The materials associated with initial planning of the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.3.3.2 - Operational GovOps Review [Core]  <!-- UUID: 3706028f-c737-4cbd-850c-d28a09c31b68 -->
+
+The materials associated with Operational GovOps Review during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.3.3.3 - Artifact Edit Proposal [Core]  <!-- UUID: ffc72f3e-e3aa-41c2-ac59-71a192764297 -->
+
+The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.4 - Osero-Ozone Instance Configuration Document [Core]  <!-- UUID: dd0515f7-64fd-411e-8dff-07fc1b2be962 -->
+
+The documents herein contain the Instance Configuration Document for the Osero-Ozone Executor Accord Primitive Instance.
+
+###### A.6.1.2.2.2.2.1.2.4.1 - Parameters [Core]  <!-- UUID: 8e2c8fc6-9195-4315-b662-db147bdcb2f8 -->
+
+The documents herein define the parameters of the Osero-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.4.1.1 - Prime Agent [Core]  <!-- UUID: 806afd09-d83f-4c8e-a512-49f64bce8587 -->
+
+The Osero Prime Agent’s details including foundation and development company are specified in [A.2.8.2.6.1.1.2 - Osero Details](https://sky-atlas.io/#d9b2da30-bdae-47a6-96ff-966f92ce2b7e).
+
+###### A.6.1.2.2.2.2.1.2.4.1.2 - Custom Instance Parameters [Core]  <!-- UUID: d9004e41-77d0-4d34-8c08-6a65e6f5da82 -->
+
+The documents herein define the custom parameters of the Osero-Ozone Instance of the Executor Accord Primitive, if any.
+
+###### A.6.1.2.2.2.2.1.2.4.2 - Operational Process Definition [Core]  <!-- UUID: 7e4a065d-fde3-431a-be3b-9af26e875123 -->
+
+The documents herein define the process for the ongoing management of the Osero-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.4.3 - Data Repository [Core]  <!-- UUID: df368129-b922-416d-ba86-4a0d202135d0 -->
+
+The documents herein contain data relevant to the Osero-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.4.3.1 - Initial Planning [Core]  <!-- UUID: 4292e536-c278-4b5f-802a-402df8ce03a4 -->
+
+The materials associated with initial planning of the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.4.3.2 - Operational GovOps Review [Core]  <!-- UUID: 0d2b1b88-e119-4a68-95c1-db4b6bdbef56 -->
+
+The materials associated with Operational GovOps Review during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.4.3.3 - Artifact Edit Proposal [Core]  <!-- UUID: 0cf4bc5f-73e9-4915-bef1-a812c06e1e58 -->
+
+The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.5 - Launch Agent 7-Ozone Instance Configuration Document [Core]  <!-- UUID: 6843c012-4dd3-4255-bb7b-667413496680 -->
+
+The documents herein contain the Instance Configuration Document for the Launch Agent 7-Ozone Executor Accord Primitive Instance.
+
+###### A.6.1.2.2.2.2.1.2.5.1 - Parameters [Core]  <!-- UUID: 28f008ca-fcc8-4120-8777-83c7968c7d5d -->
+
+The documents herein define the parameters of the Launch Agent 7-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.5.1.1 - Prime Agent [Core]  <!-- UUID: 962425c9-d982-4eae-a70d-d4ae8a57b811 -->
+
+Launch Agent 7 Prime Agent’s details will be specified in future iterations of the Atlas.
+
+###### A.6.1.2.2.2.2.1.2.5.1.2 - Custom Instance Parameters [Core]  <!-- UUID: bd095b33-87d0-4fb2-bb44-6946b139786a -->
+
+The documents herein define the custom parameters of the Launch Agent 7-Ozone Instance of the Executor Accord Primitive, if any.
+
+###### A.6.1.2.2.2.2.1.2.5.2 - Operational Process Definition [Core]  <!-- UUID: 0af39a2e-4d87-4a07-9f22-512b99c9ebee -->
+
+The documents herein define the process for the ongoing management of the Launch Agent 7-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.5.3 - Data Repository [Core]  <!-- UUID: a31b5cc7-2926-4d1d-b837-10d89ec1ed13 -->
+
+The documents herein contain data relevant to the Launch Agent 7-Ozone Instance of the Executor Accord Primitive.
+
+###### A.6.1.2.2.2.2.1.2.5.3.1 - Initial Planning [Core]  <!-- UUID: dfe6276a-4477-4f8d-a6ca-579580c22801 -->
+
+The materials associated with initial planning of the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.5.3.2 - Operational GovOps Review [Core]  <!-- UUID: 267d2086-f51d-478a-8283-994bbee3f023 -->
+
+The materials associated with Operational GovOps Review during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.2.5.3.3 - Artifact Edit Proposal [Core]  <!-- UUID: 0eac9558-762e-4cb7-a6a3-ecb7c762fc03 -->
+
+The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.
+
+###### A.6.1.2.2.2.2.1.3 - Completed Instances [Core]  <!-- UUID: 5cd43b2b-e30f-49b7-842d-e8b6d4fd4e73 -->
+
+The Instances of the Executor Accord Primitive with `Completed` Status are stored herein.
+
+###### A.6.1.2.2.2.2.2 - Root Edit Primitive [Core]  <!-- UUID: 8c89a791-c93e-4c23-a3f9-20e66b0e4194 -->
+
+The documents herein contain all data and specifications for Ozone's Instance of the Root Edit Primitive. See [A.2.2.5.2 - Root Edit Primitive](https://sky-atlas.io/#78488c6b-d77f-4344-b954-476e415a2c7d).
+
+###### A.6.1.2.2.2.2.3 - Light Agent Primitive [Core]  <!-- UUID: 25708b19-eab8-4b78-8c17-76e876137320 -->
+
+The documents herein contain all data and specifications for Ozone's Instances of the Light Agent Primitive. See [A.2.2.5.3 - Light Agent Primitive](https://sky-atlas.io/#44028423-2cd1-40cb-89ac-3f762b602b90).
+
+###### A.6.1.2.2.2.3 - Ecosystem Upkeep Primitives [Core]  <!-- UUID: 585a4460-3c5c-4ea6-80f1-47b4c411ef85 -->
+
+The documents herein implement the Ecosystem Upkeep Primitives for Ozone. See [A.2.2.6 - Ecosystem Upkeep Primitives](https://sky-atlas.io/#25673fd2-76cb-4c4d-8ec6-8c489207bcfc). 
+
+##### A.6.1.2.2.3 - Omni Documents [Core]  <!-- UUID: 0150ce2f-47b5-4b3d-9204-05995b9d3c34 -->
+
+The documents herein define Ozone's strategic intent and operational processes relating to infrastructure inherited from Sky Core, activities unrelated to Sky Primitives, or activities spanning multiple Sky Primitives.
+
+###### A.6.1.2.2.3.1 - Governance Information Unrelated To Root Edit Primitive [Core]  <!-- UUID: 7cc51758-2a4e-4f89-8bca-8ae606a7d8ce -->
+
+The documents herein specify Ozone governance information that is unrelated to the use of the Root Edit Primitive. The governance process for updating the Ozone Artifact is specified in the Root Edit Primitive above at A.6.1.2.2.2.2.2 - Root Edit Primitive.(https://sky-atlas.io/#8c89a791-c93e-4c23-a3f9-20e66b0e4194).
+
+###### A.6.1.2.2.3.1.1 - Sky Forum [Core]  <!-- UUID: 122b24b0-e135-46ce-8e29-c2167a792802 -->
+
+Ozone uses the Sky Forum for governance-related discussion. Posts should use the "Ozone Executor" category.
+
+###### A.6.1.2.2.3.1.2 - Sky Ecosystem Emergency Response [Core]  <!-- UUID: a7d20c62-a579-40d9-847c-e793a9a3cd52 -->
+
+The documents herein specify Ozone’s emergency response protocol in situations that impact the entire Sky Ecosystem. This protocol will be specified in a future iteration of the Ozone Artifact.
+
+###### A.6.1.2.2.3.1.3 - Agent-Specific Emergency Response [Core]  <!-- UUID: 5bc2da42-3c50-4794-9c78-e909d7b54554 -->
+
+The documents herein specify Ozone’s emergency response protocol in situations solely impacting Ozone versus the broader Sky Ecosystem. This protocol will be specified in a future iteration of the Ozone Artifact.
+
+###### A.6.1.2.2.3.2 - Ecosystem Accords [Core]  <!-- UUID: d327b576-15bb-4012-8e1e-734331f31a7f -->
+
+Ozone has formally agreed to the Ecosystem Accords herein.
+
+###### A.6.1.2.2.3.2.1 - Ecosystem Accord 9 [Core]  <!-- UUID: 1fccb820-73b7-4c60-9083-fc1c35041b5e -->
+
+Ozone engaged in terms of agreement with Sky in Ecosystem Accord 9, located in [A.2.8.2.9 Ecosystem Accord 9: Sky and Ozone](https://sky-atlas.io/#cb3c159b-46fd-4e85-a6d5-e2ab17977ac8)
 
 #### A.6.1.2.3 - Core Council Executor Agent 1 [Core]  <!-- UUID: 12b14e05-159b-4e71-822b-7dd2c17c3358 -->
 
